triagebot.toml

```diff
@@ -58,6 +58,17 @@ Thanks! <3
 """
 label = "O-ARM"
 
+[ping.risc-v]
+message = """\
+Hey RISC-V Group! This bug has been identified as a good "RISC-V candidate".
+In case it's useful, here are some [instructions] for tackling these sorts of
+bugs. Maybe take a look?
+Thanks! <3
+
+[instructions]: https://rustc-dev-guide.rust-lang.org/notification-groups/risc-v.html
```

Comment:
This page does not yet exist, and probably should before this is merged.

Reply:
It's already mentioned in the OP and https://github.com/rust-lang/rustc-dev-guide/pull/814 is under review.